include/boost/histogram/python/register_axis.hpp

```diff
@@ -227,27 +212,27 @@ py::class_<A> register_axis(py::module &m, const char *name, Args &&... args) {
         .def(py::self == py::self)
         .def(py::self != py::self)
 
-        .def("update", &A::update, "i"_a, "Bin and add a value if allowed")
         .def_property_readonly(
             "options",
             [](const A &self) {
                 return options{static_cast<unsigned>(self.options())};
             },
             "Return the options associated to the axis")
+
         .def_property(
             "metadata",
             [](const A &self) { return self.metadata(); },
             [](A &self, const metadata_t &label) { self.metadata() = label; },
             "Set the axis label")
 
+        .def_property_readonly(
+            "size",
```

Comment:
Note that np.ndarray.size returns a tuple, and this doesn't (since an axis is always 1D), but I think this is still fine, and matches Boost.Histogram.

Reply:
I think you confuse `.size` and `.shape`. `np.array((1,2,3)).size == 3`.

Reply:
`.size` is always a number, even for Nd arrays.

Reply:
Yes, I am confused. Ignore me, please. :)